run.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+import sys
+import time
+
+from subprocess import CalledProcessError
+
+
+def log(message):
+    print >> sys.stderr, message
+
+
+def run_command(args):
+    try:
+        subprocess.check_call(args)
+    except CalledProcessError as e:
+        log('Command failed with exit status %d: %s' % (e.returncode, ' '.join(args)))
+
+
+PROCESSES = []
+
+
+def run_command_async(args):
+    p = subprocess.Popen(args)
+    PROCESSES.append(p)
+    return p
+
+
+def cleanup_processes():
+    for p in PROCESSES:
+        try:
+            p.terminate()
+            time.sleep(0.9)
+            p.kill()
+        except:
+            pass
+
+
+def main():
+    server_args = []
+
+    run_command(['pip', 'install', '-e', '.'])
+    run_command(['./example_project/manage.py', 'migrate', '--noinput'])
```

Comment:
We'll likely need to put the python command in front of manage.py here, as on windows it doesn't know how to just execute the file. I'll test when I get a chance.